service/storage_proxy.cc

```diff
@@ -2140,35 +2140,39 @@ future<result<>> storage_proxy::mutate_end(future<result<>> mutate_result, utils
     if (lc.is_start()) {
         stats.estimated_write.add(lc.latency());
     }
-    try {
+    return utils::result_futurize_try([&] {
```

Comment:
Does this increase tasks/allocations?

Reply:
I think it shouldn't. The `try` body and `catch` handlers here all return ready futures. `utils::result_futurize_try` probably won't allocate tasks because all futures will be ready, but I'll check.

Reply:
I think the futures won't be ready since there's an execution_stage in there. Maybe you removed a continuation here too?

Reply:
`mutate_end` is always called in `then_wrapped`, and given a ready future.